### ISSUE TITLE
Add links to books in index columns

### ISSUE THREAD

**Issue by duffn** [duffn/theouterrim#11]
Each book in the index columns should link to the individual book.

<img width="407" alt="Screen Shot 2020-02-20 at 6 56 53 PM" src="https://user-images.githubusercontent.com/3457341/74997255-ea950280-5412-11ea-9843-f6041b09d213.png">

**Comment by AndreLemay** [comment]
Just as a note, to make the links between data less brittle, I'm replacing all the book names in the index data with the generatedId of the referenced book.

**Comment by AndreLemay** [comment]
@duffn Hoping you might have an answer since google is failing me.. Right now we have all our column definitions in separate files that just export an array. In order to render the links to the correct page, I need to pull the book data. I've tried googling how to use a graphql query outside of any react component... but basically every result I see is some weird case of using a graphql query outside of something (like pages) but still inside a react component. 

Basically what I want to know is if there's any way to just get the book data in a variable that I can access is just plain js without any react stuff.

If that's not possible, I think we might need to move the column definitions into the pages themselves so we can get the graphql query data inside the react component.

**Comment by duffn** [comment]
I'm not sure about that, to be honest. I've only run queries inside of a page/template or component. If it's not possible, then I suppose they'll need to go somewhere else, though they are used in both the resource page and the books page.

**Comment by AndreLemay** [comment]
Hmm... Maybe we could turn them into a react component. Like a "WeaponsColumnProvider" that would wrap the StatPage or Table components and essentially do nothing but pass the columns to the children and render them.

**Comment by duffn** [comment]
That sounds like that could be a good approach.

**Comment by duffn** [comment]
I didn’t mention it originally, but we should make the indices on the individual card as well. If that’s outside the scope of this issue, I’ll write a new one.
